pinot-tools/src/main/java/org/apache/pinot/tools/admin/PinotAdministrator.java

```diff
@@ -192,7 +192,11 @@ public static void main(String[] args) {
     PluginManager.get().init();
     PinotAdministrator pinotAdministrator = new PinotAdministrator();
     pinotAdministrator.execute(args);
-    if (System.getProperties().getProperty("pinot.admin.system.exit", "false").equalsIgnoreCase("true")) {
+    // Ignore `pinot.admin.system.exit` property for Pinot quickstarts.
+    if ((args.length > 0) && ("quickstart".equalsIgnoreCase(args[0]))) {
+      System.clearProperty("pinot.admin.system.exit");
```

Comment:
Is there side-effect of clearing a property? If so, might better to just return instead of clearing it